java/src/jmri/jmrix/dcc4pc/Dcc4PcReply.java

```diff
@@ -137,6 +137,20 @@ public int[] getDataAsArray() {
         return _dataChars.clone();
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public String toString() {
+        StringBuilder buf = new StringBuilder();
+        buf.append("0x");
+        buf.append(Integer.toHexString(0xFF & _dataChars[0]));
```

Comment:
I suggest that these two lines are changed to:
`buf.append(String.format("0x%02X", 0xFF & _dataChars[0]));`

This will ensure you have two digit hexadecimal numbers.

Same applies to lines 147-148.